src/utils/helpers/poller/poller.js

```diff
@@ -41,45 +41,49 @@ import './../../promises';
  */
 export default (queryOptions, functions, options) => {
   if (!setupValid(queryOptions, functions)) {
-    return;
+    return undefined;
```

Comment:
`return;` returns undefined, so by fixing the linting error and explicitly returning `undefined` via `return undefined;` there should be no unintended side effects.